buildSrc/src/main/kotlin/otel.publish-conventions.gradle.kts

```diff
@@ -9,8 +9,10 @@ publishing {
       plugins.withId("java-platform") {
         from(components["javaPlatform"])
       }
-      plugins.withId("java-library") {
-        from(components["java"])
+      if(project.name != "agent-for-testing") {
```

Comment:
Instead of hard-coding the project name, would it make sense to do it similarly to `mavenGroupId`? You could add some `skipJavaComponentPublish` property and only use it in the `agent-for-testing` project.

Reply:
I am not sure I want to add yet another configuration flag. Ultimate solution should be something different.